contrib/babelfishpg_tds/src/backend/tds/tds.c

```diff
@@ -893,18 +893,26 @@ TdsSetGucStatVariable(const char *guc, bool boolVal, const char *strVal, int int
 }
 
 void
-TdsSetAtAtStatVariable(const char *at_at_var, int intVal, uint64 bigintVal)
+TdsSetAtAtStatVariable(enum TdsAtAtVarType at_at_var, int intVal, uint64 bigintVal)
```

Comment:
No need to use the enum prefix now as we've defined the same using typedef.

Reply:
LGTM. Done.